### ISSUE TITLE
Integrate via lein-plugin/boot?

### ISSUE THREAD

**Issue by VictorBjelkholm** [yogthos/clojure-error-message-catalog#21]
I'm a complete beginner with Clojure (mainly focused on Clojurescript) so maybe I'm thinking completely wrong. I do appreciate this initiative since error message in Clojure (coming from a non-java background) is very foreign and hard to parse.

But, would it be possible to hook into something in lein/boot when an error happen and automatically link to something in this catalog? That would make usage of this so much easier.

**Comment by yogthos** [comment]
Some kind of lein middleware is definitely one of the ideas being considered. It could try to match the exception message with one of the recorded ones and add a link or something. First, we need to collect a few more errors though to have a minimum knowledge base.
